### DISCUSSION TITLE
Why WidgetReference has been renamed to WidgetRef?

### DISCUSSION THREAD

**Discussion by julienlebren** [rrousselGit/riverpod#560] [Q&A] [answered]
It's not a common practice to use abbreviations in class names, the first one was good, why was it so renamed?

**Comment by rrousselGit** [comment, marked as answer]
This was done to match `ref` and `ProviderRef`, which are getting fairly lengthy with `AutoDisposeStateNotifierProviderRef<StateNotifier, State>`

**Reply by julienlebren** [reply]
Ok. I don't like this but ok 😅
